### DISCUSSION TITLE
How to turn on fractional scaling

### DISCUSSION THREAD

**Discussion by zero-hugo** [elementary/triage#650] [Q&A] [answered]
The system currently only supports integer scaling, but 1x scaling is too small and 2x scaling is too large. I use text scaling to enlarge the interface, but this method is ineffective for some applications. I need fractional scaling. Can anyone tell me how to enable it? Why doesn't the system support it by default?

**Comment by vvvvvvitor** [comment]
From what I've heard there is no way to enable fractional scaling natively, your only option is to change the font size.

**Comment by leolost2605** [comment, marked as answer]
If you're on elementary OS 8 and on the wayland (secure) session you can enable fractional scaling by running
`gsettings set org.gnome.mutter experimental-features "['scale-monitor-framebuffer']"` in the terminal. Now fractional scaling options should show up per monitor in the display settings.
Note though that fractional scaling is still experimental and some apps may look blurry. I use it everyday though :)

**Reply by vvvvvvitor** [reply]
Didn't know! I'll try it out aswell
